.eslintrc

```diff
@@ -0,0 +1,17 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "extends":  [
+    "airbnb-base",
```

Comment:
This uses Airbnb base as a style. Open to alternatives on this one. I don’t really care, but I find @typescript-eslint to leave too much to interpretation still